### DISCUSSION TITLE
Higher version required for github copilot

### DISCUSSION THREAD

**Discussion by bfgelectronics** [community/community#13918] [Copilot Conversations] [answered]
Does this issue occur when all extensions are disabled?: Yes

-VS Code Version: 1.65.2
-OS Version: Arch Linux

Steps to Reproduce:

-Download the Github Copilot Extension
-Log in to the extension
-Get greated with the notification: GitHub Copilot requires VS Code 1.57+. Please update your VS Code

**Comment by davecheney** [comment, marked as answer]
Sadly Arch Linux does not come with VSCode, or comes with a recompiled version of VSCodium or OSS Code. Those variants do not currently work with Copilot. Please try installing vscode directly from the vendors website.
